ref/test/test_input/c_sw_12x24.nl

```diff
@@ -5,6 +5,7 @@
   output_file='c_sw_12x24.nc'
   log_dir='test_output'
   log_file='c_sw_12x24.log'
+  interpFactor = 5
```

Comment:
Shouldn't the default be 1?  I think it would be nice to follow the yppm example where we have the default namelist use the interp factor=1, and then we have other namelists with different interp factors.  We can't test large interp factors in CI, so I'd dial back the tests to use 2 or 3.

Reply:
Added back the original test data.  Dialed back the interpFactor to 3.